packages/imperative/src/profiles/src/utils/ProfileUtils.ts

```diff
@@ -18,6 +18,7 @@ import * as nodePath from "path";
  * Set of static utility methods to assist with creating profile option names from profile types, constructing the
  * root directory, reforming responses for different purposes, etc.
  *
+ * @internal
```

Comment:
I know we recently had a change that something like this broke Zowe Explorer. Do we see this having the same effect? Is there something Imperative exports for full conversion?

Reply:
A quick search of ZE code shows no import of ProfileUtils. ZE should use imperative's ConfigBuilder.convert() to convert V1 profiles. That function does not rely on ProfileUtils.